nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-0-11-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/ConsumerLease.java

```diff
@@ -520,6 +520,14 @@ private void writeRecordData(final ProcessSession session, final List<ConsumerRe
                     try {
                         reader = readerFactory.createRecordReader(attributes, in, logger);
                     } catch (final Exception e) {
+                        if (e instanceof IOException) {
```

Comment:
makes sense here just to catch IOException in a block and the general Exception in the following block, nO?

Reply:
Blurgh, yes, sorry. Will address & push.